src/main/controllers/test_connection.ts

```
@@ -1,20 +1,30 @@
+            }
+            
             const topicsList = await service.getTopics(kafkaBrokers);
             const topicExists = topicsList.includes(topic);
-            if (!topicExists) throw { message: "Topic does not exist" };
-            const result = { connectionEstablished: true, topicExists: topicExists }
+            if (!topicExists) throw new Error("Topic does not exist");
+            const result = { connectionEstablished: true, topicExists: topicExists };
+            response.setHeader('Content-Type', 'application/json');
             response.status(200).send(result);
         } catch (error: any) {
             console.log(error?.message);
```

ESLint

[error] Unexpected console statement.

(no-console)